README.md

```diff
@@ -1,43 +1,56 @@
 # CellProfiler-RunOnCluster 
 [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.3275888.svg)](https://doi.org/10.5281/zenodo.3275888)
 
-A CellProfiler module for submitting batch jobs to a cluster running slurm.
+A CellProfiler module for submitting batch jobs to a cluster running Slurm, tested with [CellProfiler 4.2.1 and CellProfiler 4.2.2rc](https://cellprofiler.org/releases) on Windows 10 and 11.
 
 ## Installation
 ### Windows
-
-A precompiled executable with default settings for the Sunbird cluster in Swansea is provided in [releases](https://github.com/sa2c/CellProfiler-RunOnCluster/releases/download/v1.0/CellProfiler.exe). These executables require that a Java Runtime Environment is installed. You can get one for example form the [Java downloads page](https://www.java.com/en/download/). 
+Note as our plugin requires additional libraries which aren’t packaged with CellProfiler 4, you’ll need to build CellProfiler from source rather than using a pre-packaged version. ([Reference](https://cellprofiler-manual.s3.amazonaws.com/CellProfiler-4.0.6/help/other_plugins.html?highlight=plugins))
 
 ### Source Installation
+When running on Linux or developing your own plugins, the most straight forward method is to copy the plugin sources to your Cellprofiler plugins directory. It is recommended to use the default ``CellProfiler\plugins`` folder as your plugin directory, but you can still have the plugins in other locations. Follow the instructions for installing CellProfiler on the [Wiki](https://github.com/CellProfiler/CellProfiler/wiki). Choose you operating system on the right side panel. Once you have installed CellProfiler, set the plugins directory in Cellprofiler preferences, then save and restart CellProfiler. 
+
+Some useful guidlines for developing CellProfiler plugins:
+
+[Orientation to CellProfiler code](https://github.com/CellProfiler/CellProfiler/wiki/Orientation-to-CellProfiler-code)
+
+[CellProfiler 3 to 4 migration guide](https://github.com/CellProfiler/CellProfiler-plugins/wiki/CellProfiler-3-to-4-migration-guide)
 
-When running on Linux or developing your own plugins, the most straight forward method is to copy the plugin sources to your Cellprofiler plugins directory.
+### Previous Plugins
+Please note [Previous plugins](https://github.com/sa2c/CellProfiler-RunOnCluster/archive/master.zip) do not fully work with the precompiled CellProfile executable (version 3.1.8), which has default settings for the SCW Sunbird cluster, can be found in [releases](https://github.com/sa2c/CellProfiler-RunOnCluster/releases/download/v1.0/CellProfiler.exe). These executables require that a Java Runtime Environment is installed. You can get one for example form [Java](https://www.java.com/en/download/).
```

Comment:
We shouldn't advertise something that is outdated and known to not work; remove this section.